### ISSUE TITLE
Tab completion doesn't work in Thunderbird 60

### ISSUE THREAD

**Issue by dmusican** [alainfrisch/nostalgy#56]
I'm now running Thunderbird 60.2.1, with Nostalgy 0.2.36. Whenever completing the name of a folder (say when using "S" to save), the usual dialog appears, but tab doesn't work anymore for competion; the tab key seems to do nothing. I can use the up arrow to select among the choice presented.

**Comment by klieret** [comment]
I can confirm that as well (also using 60.2.1 with Nostalgy 0.2.36).

**Comment by dmusican** [comment]
I'm no longer having this problem. I did just go into the Completion options and check "Tab triggers shell-like completion" which solved the issue.

**Comment by caugner** [comment]
Cannot reproduce this issue in 0.3 with TB 68: Having `[x] Tab triggers shell-like completion` enabled, the tab completion works as expected.

@klieret Do you still experience this issue in Nostalgy 0.3 with TB 68? If so, can you please try and disable all other extensions to rule out that one of those is preventing Nostalgy from receiving the <kbd>Tab</kbd> keyboard event?

**Comment by delsvr** [comment]
Yes, this is still happening for me with Nostalgy 0.3 on TB 68. The tab option is selected, I've restarted Thunderbird, and all other extensions were disabled, including Lightning. Have tried on both Linux and Windows.

Hitting <Enter> selects the top-rated result, so I just need to be more specific with each search.
